src/content/1.7/development/components/email-theme/_index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: The Email Theme component
+menuTitle: Email Theme
+weight: 2
+---
+
+# The Email Theme component
```

Comment:
Wouldn't it make more sense to call it `Mail Template component` to follow same naming that is in code?

Reply:
This would we logical from a developper perspective, but the page in the back office is named "Email theme" so I thought matching with this would be more "user friendly"

Reply:
Besides the word "template" is a bit confusing in this situation, I had to add a section `Vocabulary` in this doc to avoid confusion between layout and template.. not really easy to find correct and clear naming on this feature

Reply:
Well, it's developer documentation, so calling same component in different names is weird imo. It's the same as calling `Grid component` in code, but `List component` in docs.